### DISCUSSION TITLE
Support for snowflake dialect

### DISCUSSION THREAD

**Discussion by janepwang** [hibernate/hibernate-orm#4838] [Design Proposals]
While working with snowflake, there are several differences breaking the DDL JDBC statement execution.
- SQL Types and snowflake types miss match
- set index causing error
- set constraint causing error.

This might be a combo of dialect and hibernate configuration.

**Comment by beikov** [comment]
I don't know anything about Snowflake and the Hibernate team decided some months ago that new dialects must come from the vendors themselves. Please see https://github.com/hibernate/hibernate-orm/blob/main/dialects.adoc for more information.
